src/Docker/Client/Http.hs

```diff
@@ -94,7 +94,7 @@ runDockerT (opts, h) r = runReaderT (unDockerT r) (opts, h)
 mkHttpRequest :: HttpVerb -> Endpoint -> DockerClientOpts -> Maybe Request
 mkHttpRequest verb e opts = request
         where fullE = T.unpack (baseUrl opts) ++ T.unpack (getEndpoint (apiVer opts) e)
-              initialR = parseUrl fullE
+              initialR = parseUrlThrow fullE
```

Comment:
@jprider63 you are right... That's what I wanted to use but I could have sworn that it was the other way around so that's why I used parseUrlThrow. Thanks.